packages/fast-components/src/button/index.ts

```diff
@@ -0,0 +1,103 @@
+import { customElement } from "@microsoft/fast-element";
+import { Button, buttonTemplate } from "./button";
+import { Anchor, anchorTemplate } from "./anchor";
+import { ButtonStyles as styles } from "./button.styles";
+import { designSystemConsumer } from "../design-system-consumer";
+import {
+    accentFillActive,
+    accentFillHover,
+    accentFillRest,
+    accentForegroundActive,
+    accentForegroundCutRest,
+    accentForegroundHover,
+    accentForegroundRest,
+    neutralFillActive,
+    neutralFillFocus,
+    neutralFillHover,
+    neutralFillRest,
+    neutralFillStealthActive,
+    neutralFillStealthHover,
+    neutralFillStealthRest,
+    neutralFocus,
+    neutralFocusInnerAccent,
+    neutralForegroundRest,
+    neutralOutlineActive,
+    neutralOutlineHover,
+    neutralOutlineRest,
+} from "../styles/recipes";
+
+// Button
+@customElement({
+    name: "fast-button",
+    template: buttonTemplate,
+    styles,
+    shadowOptions: {
+        delegatesFocus: true,
+    },
+})
+@designSystemConsumer({
+    recipes: [
+        accentFillActive,
+        accentFillHover,
+        accentFillRest,
+        accentForegroundActive,
+        accentForegroundCutRest,
+        accentForegroundHover,
+        accentForegroundRest,
+        neutralFillActive,
+        neutralFillFocus,
+        neutralFillHover,
+        neutralFillRest,
+        neutralFillStealthActive,
+        neutralFillStealthHover,
+        neutralFillStealthRest,
+        neutralFocus,
+        neutralFocusInnerAccent,
+        neutralForegroundRest,
+        neutralOutlineActive,
+        neutralOutlineHover,
+        neutralOutlineRest,
+    ],
+})
+export class FASTButton extends Button {}
+export * from "./button";
+
+// Anchor
```

Comment:
I get that these are very similar, but why is this in the same file as a different export name? Seems hard to track down and somewhat against an extensibility model.

Reply:
+1 - I'd be for moving this to a separate "anchor" directory

Reply:
resolved